docker-compose.yml

```diff
@@ -0,0 +1,89 @@
+version: '3'
+
```

Comment:
I see you are duplicating changes. I propose to put all the common variables in the 

x-common-variables: &common-variables
  SCYLLA_URL: scylla:9042
      SCYLLA_USER: ${SCYLLA_USER}
      SCYLLA_PASSWORD: ${SCYLLA_PASSWORD}
      AWS_BUCKET_NAME: near-lake-data-mainnet
      NEAR_RPC_URL: https://archival-rpc.mainnet.near.org
      AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
      AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
      AWS_DEFAULT_REGION: ${AWS_DEFAULT_REGION}
      OTEL_EXPORTER_JAEGER_AGENT_HOST: jaeger
      OTEL_EXPORTER_JAEGER_AGENT_PORT: 6831

I'm not sure if version 3 of compose supports this use case. Maybe you need to change it to 3.4 or higher

Translated with www.DeepL.com/Translator (free version)